docs/plugin-protocol/tfplugin5.1.proto

```diff
@@ -0,0 +1 @@
+../../internal/tfplugin5/tfplugin5.proto
```

Comment:
Hmm my instinct was to invert this and make the other location be the symlink, to make it clearer that these files are considered immutable once "released".

I'm not super against this model, but I worry a bit that it will make it easier to forget to deal with the versioning concerns when updating it, and thus to create a confusing/ambiguous situation.

Reply:
Oh, that works too! It won't help with the case where someone edits the file through the symlink in `./internal/tfplugin`, but it makes the docs more of the "canonical source"

Reply:
Yeah... I was optimistically hoping that an editor would draw attention to the fact that's a symlink, but if not at least the eventual PR should show that there's an edit to a file called `tfproto5.1` rather than the creation of a new file called `tfproto5.2` and set of some :alarm_clock: :grin: